packages/sql-indexer/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@ckb-lumos/sql-indexer",
+  "version": "0.3.1",
+  "description": "SQL based CKB Indexer using typeorm",
```

Comment:
```suggestion
  "description": "SQL based CKB Indexer using knex",
```

Reply:
Remove the whole `using` part since we might rewrite it later without knex